doc/release-notes/release-notes-1.0.2.md

```diff
@@ -5,12 +5,12 @@ ITH4Coinomia (2):
 S. Matthew English (1):
       enforcing consistency 'tor' to 'Tor'
 
-Sean Bowe (2):
+Sean Bowe (1):
       Write R1CS output to file in GenerateParams.
-      1.0.2 release.
```

Comment:
This is one change I'd be happy reverting, since it would never be in the actual release notes due to the way our release process works.